recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
-{% set version = "3.6.3" %}
+{% set version = "3.7.0" %}
 
 package:
   name: geos
   version: {{ version }}
 
 source:
   url: https://github.com/libgeos/libgeos/archive/{{ version }}.tar.gz
-  sha256: 1f51069041c3df85b557f14c25c8fc3b31a1e25cdf05a13b480351a18f8da644
+  sha256: c4572cff40f52aad43bf3cccc3f81364f9e842f9b4b66ef23a28855037663f91
   patches:
     - xmltester.cpp.patch  # [win]
     - DirectedEdgeStar_vc9.patch  # [win and vc<14]
     - GEOSGeom_createCollection_vc9.patch  # [win and vc<14]
 
 build:
-  number: 1
+  number: 0
```

Comment:
@gillins the `GEOSGeom_createCollection_vc9.patch` is failing and I wonder if it is worth refreshing it... Not sure if this version of `geos` even builds on `vc9`.

What do you think? Should we drop `vc9` builds?

Reply:
> What do you think? Should we drop vc9 builds?

Given that `libgdal` no longer supports `vc9` I'd say dropping it would be a good plan. Although I can have a look if you find a good reason to continue with it...

Reply:
> Although I can have a look if you find a good reason to continue with it...

No good reason. Let's drop it!

Reply:
down with vc9!  Modern C++ forever!